### DISCUSSION TITLE
is having NAT, host-only and bridge on the same VM conflicting?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5131] [General] [answered]
VMware Workstation 16 pro
Current version - securityonion-2.3.61-STENODOCKER.iso
CentOS 7
 
Hi, I am wondering what conflicts having these networks on the same VM might cause?

**Comment by dougburks** [comment, marked as answer]
We are happy to help with issues that are specific to Security Onion. However, this question is not specific to Security Onion and is really just basic networking. Basic networking is considered a prerequisite for cybersecurity and is therefore beyond the scope of this discussion forum. Please review the [discussion forum guidelines](https://github.com/Security-Onion-Solutions/securityonion/discussions/1720), especially the section about generic questions and signal-to-noise ratio.

Thanks!

**Comment by iqworks** [comment]
HI Doug, guess this is an example of a question that might not be specific
to SO. I am wondering about this conflict as it relates to SO itself. I
should have clarified that more. thanks

 

From: Doug Burks ***@***.***> 
Sent: Thursday, August 12, 2021 6:47 AM
To: Security-Onion-Solutions/securityonion
***@***.***>
Cc: iqworks ***@***.***>; Author ***@***.***>
Subject: Re: [Security-Onion-Solutions/securityonion] is having NAT,
host-only and bridge on the same VM conflicting? (#5131)

 

We are happy to help with issues that are specific to Security Onion.
However, this question is not specific to Security Onion and is really just
basic networking. Basic networking is considered a prerequisite for
cybersecurity and is therefore beyond the scope of this discussion forum.
Please review the discussion forum guidelines
<https://github.com/Security-Onion-Solutions/securityonion/discussions/1720>
, especially the section about generic questions and signal-to-noise ratio.

Thanks!

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Security-Onion-Solutions/securityonion/discussions/5131#
discussioncomment-1163731> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ABKPV375SE6A3YX74WZQMNTT4
PGFXANCNFSM5B73EGRA> .
Triage notifications on the go with GitHub Mobile for iOS
<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&m
t=8&pt=524675>  or Android
<https://play.google.com/store/apps/details?id=com.github.android&utm_campai
gn=notification-email> .
<https://github.com/notifications/beacon/ABKPV37LFKKWJIMO3RKFFC3T4PGFXA5CNFS
M5B73EGRKYY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZG
OAAI4DUY.gif> 



-- 
This email has been checked for viruses by AVG.
https://www.avg.com
